common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -122,9 +142,12 @@ public boolean containsValue(Object value) {
      *
      * @param <V> the value type stored in the map.
      */
-    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V>,
-            Iterable<IntObjectMap.Entry<V>> {
+    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V> {
         final IntObjectMap<V> map;
+        Set<Integer> keySet;
```

Comment:
@Scottmitch do you have an example of how one can access a package private field of a private class outside of the private scope? It's not obvious to me that this is possible?